skype/skype-ps/skype/skype.md

```diff
@@ -1585,6 +1591,9 @@ The following cmdlet references are for Skype for Business and Microsoft Teams.
 ### [New-CsTeamsMeetingPolicy](New-CsTeamsMeetingPolicy.md)
 {{Manually Enter New-CsTeamsMeetingPolicy Description Here}}
 
+### [New-CsTeamsMeetingBrandingPolicy](New-CsTeamsMeetingBrandingPolicy.md)
+{{Manually Enter New-CsTeamsMeetingBrandingPolicy Description Here}}
```

Comment:
You need to copy the sentence from the SYNOPSIS to replace the placeholder text

Reply:
Done, copied short cmdlet description to skype.md